### PR TITLE
chore: Update .scalafmt.conf

### PR DESCRIPTION
**Pull request by patriknw**
Should be a decent time to do this now.

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -1,24 +1,20 @@
-version = 2.1.0
+version = 2.7.5
```
Comment (raboof) [inline .scalafmt.conf:1]: Should we go all the way to 3.0.3?

**Review by raboof — APPROVED** [review]
Now seems to be a good time, perhaps we want to go all the way to 3.0.3?

File: .scalafmt.conf (hunk 1 above)
Comment (patriknw) [reply .scalafmt.conf]: ok, I tried that, see https://github.com/lightbend/akkaserverless-java-sdk/pull/364/commits/f41a9ef1fecb9f9341bc6444773a8de585a95692 and https://github.com/lightbend/akkaserverless-java-sdk/pull/364/commits/0c6df07e9fd6337573fa512393465ec5861a0caa

File: testkit/src/main/scala/com/akkaserverless/javasdk/testkit/impl/TestKitValueEntityContext.scala [hunk 2]
```diff
@@ -20,8 +20,7 @@ import com.akkaserverless.javasdk.ServiceCallFactory
 import com.akkaserverless.javasdk.valueentity.ValueEntityContext
 
 /**
- * INTERNAL API
- * Used by the generated testkit
+ * INTERNAL API Used by the generated testkit
```
Comment (raboof) [inline testkit/src/main/scala/com/akkaserverless/javasdk/testkit/impl/TestKitValueEntityContext.scala:23]: Hmm, perhaps use an extra newline? Not important though.

**Review by raboof — APPROVED** [review]

File: testkit/src/main/scala/com/akkaserverless/javasdk/testkit/impl/TestKitValueEntityContext.scala (hunk 2 above)
Comment (patriknw) [reply testkit/src/main/scala/com/akkaserverless/javasdk/testkit/impl/TestKitValueEntityContext.scala:23]: yeah, that wasn't so helpful formatting, but not important

**Review by octonato — APPROVED** [review]

**Pull request merged**